licenses/management/commands/publish.py

```diff
@@ -131,7 +131,7 @@ def run_django_distill(self):
 
         self.stdout.write(f"\n{hostname}:{output_dir}")
         save_url_as_static_file(
```

Comment:
I'm wondering if using named arguments here would make the code more readable. I.e. I don't know at a glance what is `False` in the function call.